Dockerfile

```diff
@@ -1,24 +1,42 @@
-FROM amazoncorretto:11-al2-jdk
+FROM ubuntu:22.04
```

Comment:
May I know why ubuntu is in use instead of a smaller image of amazoncorretto11?

Reply:
I know the answer to this question:

`amazoncorretto` image based on Linux2 which doesn't support the installation of Node 18's dependency called `glibc 2.28`; hence cannot use `node18` on `amazoncorretto` image:

ref:

- https://repost.aws/questions/QUvkkhY--uTiSDkS6R1jFnZQ/node-js-18-on-amazon-linux-2
> Unfortunately Amazon Linux2 does not support Nodejs18.

- https://serverfault.com/questions/1132783/how-can-i-get-node18-on-amazonlinux2
> You cannot run the binary on your system because it relies on a version of glibc which is not available on your system.

understanding there is config for v18 in `package.json` but that is for React not Node

`"react": "18.1.0"`

if do not want to upgrade node engine then can just simply use `amazoncoretto` image; but there are benefits of using Node 18 to be considered:

https://www.mhtsai.me/blog/top-5-differences-between-node-16-and-node-18/